### ISSUE TITLE
Move the rakugo singleton in godot

### ISSUE THREAD

**Issue by theludovyc** [rakugoteam/rakugo-docs#28]
With godot4 we can update the internal doc. So we can move the rakugo singleton doc inside Godot. And keep rkscript doc here. We can add a link from the internal doc to rkScript doc here.

**Comment by Jeremi360** [comment]
Yes and No, as we need more then just Rakugo singleton and rkScript doc.
We need doc about Export settings, workarounds for rkScripts,
Tutorial "How to use Rakugo Vars in GDScript", RakuScriptDialogue Node.
I think we should have Rakugo Singelton doc in both: online and in godot.
